app/Services/StandService.php

```diff
@@ -475,6 +577,45 @@ public function getAircraftEligibleForArrivalStandAllocation(): Collection
         )->get();
     }
 
+    private function getDepartureStandsToAssign(): Collection
+    {
+        return NetworkAircraft::join('aircraft_stand', 'network_aircraft.callsign', '=', 'aircraft_stand.callsign')
+            ->join('stands', 'aircraft_stand.stand_id', '=', 'stands.id')
+            ->join('airfield', 'airfield.id', '=', 'stands.airfield_id')
+            ->leftJoin('stand_assignments', 'network_aircraft.callsign', '=', 'stand_assignments.callsign')
+            ->whereRaw('aircraft_stand.stand_id <> stand_assignments.stand_id')
+            ->orWhereNull('stand_assignments.stand_id')
+            ->whereRaw('airfield.code = network_aircraft.planned_depairport')
+            ->select(['network_aircraft.*', 'aircraft_stand.stand_id'])
+            ->get();
+    }
+
+    private function getDepartureStandsToUnassign(): Collection
+    {
+        return StandAssignment::join('network_aircraft', 'network_aircraft.callsign', '=', 'stand_assignments.callsign')
+            ->join('stands', 'stand_assignments.stand_id', '=', 'stands.id')
+            ->join('airfield', 'stands.airfield_id', '=', 'airfield.id')
+            ->leftJoin('aircraft_stand', 'network_aircraft.callsign', '=', 'aircraft_stand.callsign')
+            ->whereRaw('airfield.code = network_aircraft.planned_depairport')
+            ->whereNull('aircraft_stand.callsign')
+            ->select('stand_assignments.*')
```

Comment:
Same here

Reply:
Happy to chat about response on previous one :)